docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -18,6 +18,8 @@ Note that this command is not supported for the remote client.
 
 Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and will be placed in the current working directory.
 
+Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd. Alternatively, use `-Z` switch when running `mv` or `cp`.
+
```

Comment:
I'm not a fan of the words 'you' or 'your' in a man page.  Could we instead say:
```
Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on the  SELinux setup, changes to the generated files using `restorecon`, `chcon`, or `semanage` may be required to allow systemd to work on the system. Alternatively, use the `-Z` option when running `mv` or `cp`.
```

Reply:
missing 'the' in front of "`-Z` switch"

Reply:
> I'm not a fan of the words 'you' or 'your' in a man page.

There are precedents, just [a search for "note" in docs](https://github.com/search?q=Note+repo%3Acontainers%2Flibpod+path%3A%2Fdocs%2Fsource%2Fmarkdown&type=Code&ref=advsearch&l=&l=) reveals this:

> Note: You can also override the default path of the authentication file by setting the **REGISTRY_AUTH_FILE**

But of course, if you insist.

Reply:
Yes please make the change, and if you could, please change the REGISTRY_AUTH_FILE one also,

Reply:
Done. I made a slight change:
```diff
- ... may be required to allow systemd to work on the system.
+ ... may be required to allow systemd to access these files.
```

I hope it's ok?